healthz/checks/checks.go

```diff
@@ -0,0 +1,59 @@
+package checks
```

Comment:
Added `test-style` target as in Helm and this script. Looks like we're okay with licenses now. :white_check_mark: